packages/flutter_tools/lib/src/test/bootstrap.dart

```diff
@@ -0,0 +1,185 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
This needs the original copyright year

Reply:
The year that is attached to the original file?